lib/src/Exceptions.dart

```diff
@@ -0,0 +1,23 @@
+class DuplicatedException implements Exception {
+  final dynamic message;
+
+  DuplicatedException([this.message]);
+
+  String toString() {
+    if (message == null) return "Provide the message inside the exception";
+    return "DuplicatedRouteException: $message";
+  }
+}
+
+class RequestHandlerException implements Exception {
+  final dynamic message;
+
+  RequestHandlerException(
+      [this.message =
+          "Request handler should have genric type DioAdapter or MainInterceptor"]);
```

Comment:
Typo in _genric_, should be - _generic_.

Reply:
Also, filenames should be written in lowercase, in this case, as such - `exceptions.dart`

Reply:
This isn't resolved as you didn't rename the file.